test/Spec.hs

```diff
@@ -56,7 +58,82 @@ validConcurrencySpec = do
     -- It used to work with the test features that were moved into the executable since
     -- we wanted to split the project.
     cardanoFeaturesInit :: AllFeaturesInitFunction
-    cardanoFeaturesInit = \_ _ -> pure []
+    cardanoFeaturesInit = \_ _ ->
+        -- We have to be careful about the number of features here.
+        replicateM 5 cardanoFeatureGeneratorIO
+
+    -- | @CardanoFeature@ generator.
+    cardanoFeatureGeneratorIO :: IO CardanoFeature
+    cardanoFeatureGeneratorIO = do
+        cardanoFeatureName          <- generate genSafeText
+
+        let cardanoFeatureStart :: forall m. (MonadIO m) => m ()
+            cardanoFeatureStart = generateSingleCommand
+
+        let cardanoFeatureShutdown :: forall m. (MonadIO m) => m ()
+            cardanoFeatureShutdown  = generateSingleCommand
+
+        pure $ CardanoFeature
+            cardanoFeatureName
+            cardanoFeatureStart
+            cardanoFeatureShutdown
+      where
+        -- | Generate random ascii string
+        genSafeText :: Gen Text
+        genSafeText = strConv Lenient <$> listOf1 arbitraryASCIIChar
+
+-- | The function that generates a single command in @IO@.
+generateSingleCommand :: (MonadIO m) => m ()
+generateSingleCommand = do
+    generatedCommand <- liftIO $ generate concurrencyDSLGenerator
+    interepretConcurrencyDSL generatedCommand
+
+-- | The function that creates a list of commands in @IO@.
+generateListOfCommands :: (MonadIO m) => m ()
```

Comment:
This function as well as `concurrencyDSLSequenceGenerator` is not being used. Are we planning to use them on future?